std::os::wasi::io::AsFd regression on Rust 1.64

<!--
Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
along with any information you feel relevant to replicating the bug.
-->

I tried this code:

```rust
#![allow(unused_imports)]

use std::os::wasi::io::AsFd;
```

I expected to see this happen: it compiles

Instead, this happened:

With Rust 1.63, it compiles. With Rust 1.64 and Rust nightly, it gets this error:
```
error[E0658]: use of unstable library feature 'wasi_ext'
 --> src/lib.rs:3:5
  |
3 | use std::os::wasi::io::AsFd;
  |     ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: see issue #71213 <https://github.com/rust-lang/rust/issues/71213> for more information
```

--- a/library/std/src/os/wasi/io/mod.rs
+++ b/library/std/src/os/wasi/io/mod.rs
@@ -1,6 +1,6 @@
 //! WASI-specific extensions to general I/O primitives.
 
-#![unstable(feature = "wasi_ext", issue = "71213")]
+#![stable(feature = "io_safety", since = "1.63.0")]
 
 #[stable(feature = "io_safety", since = "1.63.0")]
 pub use crate::os::fd::*;

--- a/library/std/src/os/wasi/io/mod.rs
+++ b/library/std/src/os/wasi/io/mod.rs
@@ -2,5 +2,5 @@
 
 #![unstable(feature = "wasi_ext", issue = "71213")]
 
-#[unstable(feature = "wasi_ext", issue = "71213")]
+#[stable(feature = "io_safety", since = "1.63.0")]
 pub use crate::os::fd::*;

